### ISSUE TITLE
Logstash 5.2.cannot install logstash-filter-anonymize in an offline environment

### ISSUE THREAD

**Issue by rodgermoore** [logstash-plugins/logstash-filter-anonymize#11]
Ubuntu 14.04 LTS

**This works fine:**
````
sudo /usr/share/logstash/bin/logstash-plugin install logstash-filter-anonymize
Validating logstash-filter-anonymize
Installing logstash-filter-anonymize
Installation successful
````

**This gives an error:**
````
sudo /usr/share/logstash/bin/logstash-plugin prepare-offline-pack logstash-filter-anonymize
ERROR: Cannot create the offline archive, message: Cannot find plugins matching: `logstash-filter-anonymize`
````

What am I doing wrong?

**Comment by rodgermoore** [comment]
The Elastic forums as well as this Github issue tracker seems to be a big void 👎  I cannot install logstash-filter-anonymize in an offline environment which makes the upgrade to ELK5 on our production environment impossible...

**Comment by ph** [comment]
@rodgermoore There was an issue with 5.2, which was fixed in a latest 5.2.x release, if you cd in the directory before running the `logstash-plugin install` it should solve the problem.

original issue: https://github.com/elastic/logstash/issues/6639

**Comment by rodgermoore** [comment]
Fixed, thanks @ph 👍
